### PR TITLE
Changes for mobile compatibility

### PR DESCRIPTION
**Pull request by JuwonTiwo**
Calendar commented out
Changed video for mobile to gif 
Fixed menu functionality for mobile 
Changed logo to png for compatibility reasons

### CODE REVIEW

File: _sass/_header.scss [hunk 1]
```diff
@@ -53,28 +53,28 @@ body {
...
-   
-    @include mobile {
-        -webkit-transform: translateX(0%);
-        -moz-transform: translateX(00%);
-        -ms-transform: translateX(0%);
-        -o-transform: translateX(0%);
-        transform: translateX(0%);
-    }
-}
+// #menu_toggle {
```
Comment (behaghel) [inline _sass/_header.scss]: if we don't need it anymore let's delete it.

File: _includes/header.html [hunk 2]
```diff
@@ -25,4 +24,17 @@ <h1 class="dev-logo">
     {% endif %}
     {% endfor %}
   </ul>
-</section>
\ No newline at end of file
+</section>
+
+<script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
```
Comment (behaghel) [inline _includes/header.html]: Not sure I would recommend introducing jquery for simple logic like this one.

File: meetups.html [hunk 3]
```diff
@@ -47,8 +47,8 @@
           <h3><a href="http://www.devoxx.co.uk/">Devoxx UK 2015</a></h3>
           <p><b>Tom Maule</b> talks about the scalability challenges NOW TV faces with linear streaming, with reference to large sporting events and the popular fantasy drama; Game of Thrones.</p>
       </div>
-      <div class="skycom-6 video_wrapper" style="height:340px">
-        <div data-parleys-presentation="now-tv-linear-streaming-unpredictable-scalability-challenge" style="width:560px;height:315px">
+      <div class="skycom-6 video_wrapper" style="height:315px">
```
Comment (behaghel) [inline meetups.html]: do we need those `style` attribute here vs proper CSS rules?

**Comment by JuwonTiwo** [comment]
Addressed Hubert's comments

**Comment by behaghel** [comment]
Good job!

**Pull request merged**